.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,52 @@
+name: Cloud Formation Template
+
+on:
+  push: {branches: master}
+  pull_request: {branches: main} 
+  
+jobs:
+  cfn-lint-chk:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Cloud Formation Linter with Latest Version
+        uses: scottbrenner/cfn-lint-action@v2
+      - name: Cloud Formation Version & Verification - Ubuntu.
+        run: |
+          cfn-lint ./templates/nginx-plus-app-protect-ubnt1804-dev.template.yaml --ignore-checks W
+          cfn-lint --version
+  cft-upload:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - name: Install Dependencies
+        run: |
+          python -m pip install --user boto3
+          python -m pip install --user requests
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{secrets.AWS_ACCESS_KEY_ID}}
+          aws-secret-access-key: ${{secrets.AWS_SECRET_ACCESS_KEY}}
+          aws-session-token: ${{secrets.AWS_SESSION_TOKEN}}
+          aws-region: ${{secrets.AWS_REGION}}
```

Comment:
Likely AWS creds are avaliable as env vars via github secrets, therefore, this part is redundant. Please check and remove if so.